target/linux/ramips/dts/mt7621_ubiquiti_edgerouterx.dtsi

```diff
@@ -25,31 +25,38 @@
 
 &gmac0 {
 	mtd-mac-address = <&factory 0x22>;
+	label = "dsa";
 };
 
 &switch0 {
 	ports {
 		port@0 {
 			status = "okay";
-			label = "wan";
-			mtd-mac-address = <&factory 0x22>;
-			mtd-mac-address-increment = <1>;
+			label = "eth0";
```

Comment:
@dengqf6 
FYI: https://patchwork.ozlabs.org/patch/1210378/

However, with the new situation, I think the way you do it is exactly the right one.

Reply:
You can add an "Acked-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>" for this patch if you like.